docs/development/tests.rst

```diff
@@ -17,14 +17,26 @@ environments by running::
 
     tox
 
-In order to run all test including the search tests, include `"'--including-search'"`
-argument::
+By default, tox won't run tests from search,
+in order to run all test including the search tests,
+you need to override tox's posargs.
+If you don't have any additional arguments to pass,
+you can also set the ``TOX_POSARGS`` environment variable to an empty string::
 
-    tox "'--including-search'"
+    TOX_POSARGS='' tox
+
+.. note::
+
+   If you need to override tox's posargs, but you still don't want to run the search tests,
+   you need to include ``-m 'not search'`` to your command::
+
+       tox -- -m 'not search' -x
 
 .. warning::
 
    Running tests for search needs an Elasticsearch :ref:`instance running locally <development/search:Installing and running Elasticsearch>`.
+   If you are testing inside :ref:`docker compose <development/standards:Working with Docker Compose>`,
+   you'll need to override the ``ES_HOSTS`` and ``ELASTICSEARCH_DSL`` settings to point to ``search:9200`` instead.
```

Comment:
Why this is not set by default when running inside docker?

Reply:
We use the same settings file to run tests outside and inside docker, so if we change that we need to modify the hosts file on travis (not sure if we can do that in a simple way).

Do we have a way of knowing if we are running inside docker?

Reply:
We use the same settings file to run tests outside and inside docker, so if we change that we need to modify the hosts file on travis (not sure if we can do that in a simple way).

Do we have a way of knowing if we are running inside docker?

Reply:
Looks like we can set a custom host on travis https://docs.travis-ci.com/user/hosts/

Reply:
Are we ok with setting the defaults to `search:9200`?